marketplace_builder/pages/getting-started/authorization-policy/associating-authorization-policy-formconfiguration.liquid

```diff
@@ -0,0 +1,73 @@
+---
+converter: markdown
+metadata:
+  title: Associating an Authorization Policy with Form Configuration
+slug: get-started/authorization-policy/associating-authorization-policy-formconfiguration
+---
+
+This guide will help you associate an Authorization Policy with Form Configuration. 
+
+## Requirements
+So that you can follow the steps in this tutorial, you should understand the concept of Authorization Policy, and you should have created an Authorization Policy. 
+
+* [Authorization Policy](/get-started/authorization-policy/authorization-policy) 
+* [Adding an Authorization Policy](/get-started/authorization-policy/adding-authorization-policy)
+
+## Steps 
+
+Associating an Authorization Policy with Form Configuration is a two-step process:
+
+1. Create Form Configuration
+2. Add `authorization_policies` key to Form Configuration
+
+### Step 1: Create Form Configuration
+Assuming you have a simple form that allows to change the user's last name, the configuration file can look like this:
+
+{% raw %}
+
+```liquid
+---
+name: update_last_name
+resource: User
+configuration:
+  last_name:
+    validation:
+      presence: {}
+---
+{% form url: "/api/users/{{ current_user.id }}" %}
```

Comment:
I propose to just `...` everything below `---` - it will make our lifes easier later on, since this every line is deprecated :)